### PR TITLE
[RHACS] Added a note about proxy configuration for Helm and Operator

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For  https://issues.redhat.com/browse/OSDOCS-2749

Preview:
- Operator: https://deploy-preview-38778--osdocs.netlify.app/openshift-acs/latest/installing/install-ocp-operator.html#install-central-operator_install-ocp-operator
- Helm: https://deploy-preview-38778--osdocs.netlify.app/openshift-acs/latest/installing/installing_helm/install-helm-quick.html#installing-quickly_acs-install-helm-quick

Applies to:
- 3.65
- 3.66
- 3.67

<hr>

NOTE:

- This PR is for RHACS docs.
- It doesn't require any special labels or milestones.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@SimonBaeumer 
1. Can you please check if this is correct? 
2. Can we include a sample installation command that sets up proxy configuration and installs ACS, similar to the following:
   ```
   $ helm install -n stackrox \
   --create-namespace stackrox-central-services rhacs/central-services \
   --set imagePullSecrets.allowNone=true
   ```

File: modules/acs-quick-install-using-helm.adoc [hunk 1]
```diff
@@ -38,6 +38,22 @@ $ helm install -n stackrox \
   --set imagePullSecrets.allowNone=true
 ----
```
Comment (IAmATeaPot418) [inline modules/acs-quick-install-using-helm.adoc:40]: Or bypass these services in the proxy.

File: modules/install-central-operator.adoc [hunk 2]
```diff
@@ -10,7 +10,18 @@ The main component of {product-title} is called Central. You can install Central
 
 [IMPORTANT]
 ====
-When you install {product-title} for the first time, you must first install the `Central` custom resource because the `SecuredCluster` custom resource installation is dependent on certificates that Central generates.
```
Comment (IAmATeaPot418) [inline modules/install-central-operator.adoc:13]: OpenShift comes with an out of the box proxy that we integrate with and leverage. So this setting does not apply entirely and all pages to this page can be dropped.

File: modules/acs-quick-install-using-helm.adoc [hunk 3]
```diff
@@ -38,6 +38,22 @@ $ helm install -n stackrox \
...
 
+[IMPORTANT]
+====
+If you are installing {product-title} in a cluster that requires a proxy  to connect to external services, you must specify your proxy configuration by using the `proxyConfig` parameter. For example:
```
Comment (SimonBaeumer) [inline modules/acs-quick-install-using-helm.adoc:43]: nit: empty space

```suggestion
If you are installing {product-title} in a cluster that requires a proxy to connect to external services, you must specify your proxy configuration by using the `proxyConfig` parameter. For example:
```

File: modules/install-central-operator.adoc (hunk 2 above)
Comment (SimonBaeumer) [reply modules/install-central-operator.adoc:13]: If you are referring to the cluster-wide proxy this does not allow setting the proxy for individual components as it sets it the proxy config on all pods in a cluster.
WDYTH about documenting both ways and recommending the cluster-wide proxy?
And is it possible to hide the customize config in our CRs behind a button if the cluster-wide proxy is recommended? 

Here are the docs to the cluster-wide proxy:
https://docs.openshift.com/container-platform/4.9/networking/enable-cluster-wide-proxy.html

Comment (IAmATeaPot418) [reply modules/install-central-operator.adoc:13]: @gaurav-nelson - For the proxy settings this is reasonably simple. Everything that you added here should be removed. Then we'd call out that RHACS is setup to respect the cluster proxy and point them to the cluster wide proxy documentation that @SimonBaeumer is mentioning.

**Review by lpettyjo — COMMENTED** [review]
I notice that you have two commits instead of just one, so you need squash this.

Otherwise, LGTM.

**Pull request merged**